ci/README.md

```diff
@@ -0,0 +1,27 @@
+# Setting up a new infrastructure for the lambda
+
+In the terminal run following commands one by one. You need to do it every time you start a new session.
+
+'''
```

Comment:
Should be backticks (```) rather than single quotes (''') - appreciate they can look very similar in the docs! :)

Reply:
Yeah, I figured it out, thanks. Didn't expect you to look at it yet